sql/driver/conn.go

```diff
@@ -53,7 +55,18 @@ func (c *conn) Exec(stmt string, args []driver.Value) (driver.Result, error) {
 	if err != nil {
 		return nil, err
 	}
-	return driver.RowsAffected(len(result.Rows)), nil
+	switch t := result.GetUnion().(type) {
+	case nil:
+		return nil, nil
+	case *Response_Result_DDL_:
```

Comment:
Actually I think we'll almost never know. In the end state of online schema change, DDL operations will probably be asynchronous, and things like CREATE TABLE don't make sense as having affected a given number of rows.